it/helpers/application/IntegrationApplication.scala

```diff
@@ -1,40 +1,53 @@
 
 package helpers.application
 
+import connectors.{DefaultEtmpConnector, DefaultGovernmentGatewayAdminConnector, DefaultTaxEnrolmentsConnector, EtmpConnector, GovernmentGatewayAdminConnector, TaxEnrolmentsConnector}
 import helpers.wiremock.WireMockConfig
+import metrics.{DefaultServiceMetrics, ServiceMetrics}
 import org.scalatest.TestSuite
 import org.scalatestplus.play.guice.GuiceOneServerPerSuite
 import play.api.Application
 import play.api.inject.guice.GuiceApplicationBuilder
 import play.api.libs.ws.{WSClient, WSRequest}
+import uk.gov.hmrc.http.HttpClient
+import uk.gov.hmrc.play.bootstrap.http.DefaultHttpClient
+import play.api.inject.{bind => playBind}
 
 trait IntegrationApplication extends GuiceOneServerPerSuite with WireMockConfig {
   self: TestSuite =>
 
   val currentAppBaseUrl: String = "business-customer"
-  val testAppUrl: String        = s"http://localhost:$port"
+  val testAppUrl: String = s"http://localhost:$port"
 
   lazy val ws: WSClient = app.injector.instanceOf[WSClient]
 
   val appConfig: Map[String, Any] = Map(
-    "application.router"                                  -> "testOnlyDoNotUseInAppConf.Routes",
-    "microservice.metrics.graphite.host"                  -> "localhost",
-    "microservice.metrics.graphite.port"                  -> 2003,
-    "microservice.metrics.graphite.prefix"                -> "play.business-customer.",
-    "microservice.metrics.graphite.enabled"               -> true,
-    "microservice.services.etmp-hod.host"                 -> wireMockHost,
-    "microservice.services.etmp-hod.port"                 -> wireMockPort,
-    "metrics.name"                                        -> "business-customer",
-    "metrics.rateUnit"                                    -> "SECONDS",
-    "metrics.durationUnit"                                -> "SECONDS",
-    "metrics.showSamples"                                 -> true,
-    "metrics.jvm"                                         -> true,
-    "metrics.enabled"                                     -> true
+
```

Comment:
Is there a need for this val appConfig now that there is nothing within it? Shows as no usages found
I removed it recompiled and ran the tests and nothing broke

Reply:
Removed as not used